terraform-registry-manifest.json

```
@@ -0,0 +1,8 @@
+{
+    "version": 1,
+    "metadata": {
+        "protocol_versions": [
+            "6.0"
```

You'll want 5.0 for now, since this provider is being developed using terraform-plugin-sdk/v2. Otherwise, this provider will not show up for Terraform CLI 0.12-0.15 discovery. 👍 

```suggestion
            "5.0"
```